Firestore/core/src/firebase/firestore/model/field_value.mm

```diff
@@ -18,6 +18,14 @@
 
 #import "Firestore/Source/Model/FSTFieldValue.h"
 
+namespace firebase {
+namespace firestore {
+namespace model {
```

Comment:
Hah, nice...

Reply:
Yeah this was one of the clues that `using` in headers was not working as expected.